packages/web-component-tester/test/retry-failures-reporter.ts

```diff
@@ -0,0 +1,161 @@
+/**
+ * @license
+ * Copyright (c) 2018 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt The complete set of authors may be found
+ * at http://polymer.github.io/AUTHORS.txt The complete set of contributors may
+ * be found at http://polymer.github.io/CONTRIBUTORS.txt Code distributed by
+ * Google as part of the polymer project is also subject to an additional IP
+ * rights grant found at http://polymer.github.io/PATENTS.txt
+ */
+
+import * as chalk from 'chalk';
+import {spawnSync} from 'child_process';
+import * as Mocha from 'mocha';
+
+const retryTargets: string[] =
+    JSON.parse(process.env['TEST_RETRY_TARGETS'] || '[]');
+const retryTargetMax =
+    parseInt(process.env['TEST_RETRY_TARGET_MAX'] + '', 10) || 0;
```

Comment:
why the `+ ''`?

Reply:
Oh derp.  I didn't mean to leave that in.  I was worried an undefined value would throw but parseInt handles undefined just fine.  Removed the unnecessary `+ ''` concatenation.